frontend/src/components/header/index.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import Popup from "reactjs-popup";
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import { ORG_URL, ORG_NAME, API_URL } from '../../config';
+import logo from '../../assets/img/main-logo.svg';
+import { LinkIcon } from '../svgIcons';
+import { Dropdown } from '../dropdown';
+import { Button } from '../button';
+import { BurgerMenu } from './burgerMenu';
+
+
+const menuItems = [
+  {label: messages.exploreProjects, link: "/contribute"},
+  {label: messages.howItWorks, link: "/learn"},
+  {label: messages.about, link: "/about"},
+  {label: messages.help, link: "/help"}
+];
+
+class Header extends React.Component {
+  menuItems = menuItems;
+  linkCombo = "link ph3 barlow-condensed blue-dark f4 ttu";
+
+  renderMenuItems() {
+    return(
+      <div className="v-mid">
+        {this.menuItems.map((item, n) =>
+          <Link to={item.link} key={n} className={ this.linkCombo }>
+            <FormattedMessage {...item.label} />
+          </Link>
+        )}
+      </div>
+    );
+  }
+
+  renderPopupItems() {
+    return(
+      <div className="v-mid tc">
+        {this.menuItems.map((item, n) =>
+          <p key={n}>
+            <Link to={item.link} className={ this.linkCombo }>
+              <FormattedMessage {...item.label} />
+            </Link>
+          </p>
+        )}
+        <a href={`${API_URL}auth/login?redirect_to=/login/`} className="mh1 mv2 dib">
+          <Button className="bg-red white"><FormattedMessage {...messages.logIn}/></Button>
+        </a>
+        <Button className="bg-blue-dark white mh1 mv2 dib"><FormattedMessage {...messages.signIn}/></Button>
```

Comment:
Should we fix messages.signIn -> messages.joinMessage? or signUp as in other comment.